### PR TITLE
License change : Mozilla Public License Version 2.0

### PR DESCRIPTION
**Pull request by PonteIneptique**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -14,13 +14,31 @@
...
+.. image:: https://img.shields.io/badge/License-MPL%202.0-brightgreen.svg
+    :alt: License: MPL 2.0
+    :target: https://opensource.org/licenses/MPL-2.0
+
+
+.. image:: https://raw.githubusercontent.com/Capitains/tutorial-nemo/master/header.png
+    :alt: Nemo Banner
+    :target: http://capitains.org
+
+Capitains Nemo is an User Interface built around the need to make CapiTainS a easy to use, human readable standard for texts.
```
Comment (balmas) [inline README.rst]: "an easy-to-use, human-readable"

File: README.rst [hunk 2]
```diff
@@ -14,13 +14,31 @@
...
+Capitains Nemo counts multiple language implementation, including this one in Python for Flask. Built as a classic Flask
```
Comment (balmas) [inline README.rst]: multiple language implementations

**Review by balmas — APPROVED** [review]
seems fine. I made a couple of comments on english typos in the readme, in case you want to fix them.

**Comment by PonteIneptique** [comment]
Thanks ! Will merge upon Travis Feedback

**Pull request merged**